src/packages/docker-compile/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+ARG uid=1000
+ARG gid=1000
+
+RUN set -e
+
+ENV JRUBY_VERSION="1.7.27"
+
+RUN apt-get update && \
+    apt-get install -y curl software-properties-common gawk
+
+RUN apt-get install -y openjdk-8-jdk-headless build-essential git unzip debhelper && \
+    update-ca-certificates -f
```

Comment:
merge to previous install to avoid extra layers. update-ca-certificates -f is unnecessary on Ubuntu 16.04

Reply:
removed all of extra layers and removed update-ca-cerfificates